package.json

```diff
@@ -284,6 +285,28 @@
             "default": false,
             "description": "Disable the running of SourceKit-LSP.",
             "order": 6
+          },
+          "sourcekit-lsp.supported-languages": {
+            "type": "array",
+            "default": [
+              "swift",
+              "objective-c",
+              "objective-cpp",
+              "c",
+              "cpp"
+            ],
+            "description": "List of languages supported by SourceKit-LSP. This is used to determine when to start the SourceKit-LSP server.",
+            "items": {
+              "type": "string",
+              "enum": [
+                "swift",
+                "objective-c",
+                "objective-cpp",
+                "c",
+                "cpp"
+              ]
+            },
+            "order": 7
```

Comment:
Can we change the order so this appears just before `sourcekit-lsp.support-c-cpp`